mjpc/planners/gradient/planner.cc

```diff
@@ -475,6 +479,7 @@ void GradientPlanner::GUI(mjUI& ui) {
        "Zero\nLinear\nCubic"},
       {mjITEM_SLIDERINT, "Spline Pts", 2, &policy.num_spline_points, "0 1"},
       // {mjITEM_SLIDERNUM, "Spline Pow. ", 2, &timestep_power, "0 10"},
+      {mjITEM_SLIDERINT, "Deriv. Skip", 2, &derivative_skip_, "0 64"},
```

Comment:
Maybe use smaller value for the max here? Seems like so far no model works with values higher than 4-5.

Reply:
Set to max to 16.